sitemap.xml

```diff
@@ -2,7 +2,7 @@
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:news="http://www.google.com/schemas/sitemap-news/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml" xmlns:mobile="http://www.google.com/schemas/sitemap-mobile/1.0" xmlns:image="http://www.google.com/schemas/sitemap-image/1.1" xmlns:video="http://www.google.com/schemas/sitemap-video/1.1">
 <url> <loc>https://cloud.redhat.com/insights</loc> </url>
 <url> <loc>https://cloud.redhat.com/ros</loc> </url>
-<url> <loc>https://cloud.redhat.com/sample</loc> </url>
+<url> <loc>https://cloud.redhat.com/ros</loc> </url>
 <url> <loc>https://cloud.redhat.com/oops</loc> </url>
 <url> <loc>https://cloud.redhat.com/no-permissions</loc> </url>
 </urlset>
```

Comment:
just a small nitpick, new line at the end.

Reply:
@apuntamb, 
With the help of `sitemap-builder` it auto creates sitemap.xml file.